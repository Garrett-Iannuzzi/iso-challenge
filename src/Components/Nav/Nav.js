import React from 'react';
import './Nav.scss';
import { Link } from 'react-router-dom';
import images from '../../images/images.js';

const Nav = () => {
  return(
    <div>
      <nav>
        <h1>NBA IS<img src={images.basketball} className='img-bball'/>-Challenge</h1>
        <div className='btn-div'>
          <Link to='/rules'>
            <button className='btn'>Rules Expert</button>
          </Link>
          <Link to='/'>
            <button className='btn'>Home</button>
          </Link>
        </div>
      </nav>
    </div>
  )
}

export default Nav;