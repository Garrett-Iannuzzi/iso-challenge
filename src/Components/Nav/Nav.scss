nav {
  border-bottom: solid 5px black;
  display: flex;
  background-image: url('../../images/nav-img.jpg');
  background-color:rgba(0, 0, 0, 0.5);
  height: 12em;
  background-size: cover;
  justify-content: space-between;
}

.btn {
  margin: 15px;
  height: 38px;
  border-radius: 8px;
  border: solid 1px black;
  font-size: 1.5em;
  width: 80%;
  &:hover {
    cursor: pointer;
    background-color: lightpink;
    color: white;
  }
}

.img-bball {
  height: 50px;
  width: 50px;
}

h1 {
  color: white;
  font-size: 4em;
  padding-left: 10px;
}