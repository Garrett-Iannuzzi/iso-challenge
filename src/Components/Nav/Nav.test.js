import React from 'react';
import Nav from './Nav';
import { shallow } from 'enzyme';

describe('Nav', () => {

  let wrapper

  describe('Nav Component', () => {

    beforeEach(() => {
      wrapper = shallow(<Nav />)
    });

    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    });
  });

})