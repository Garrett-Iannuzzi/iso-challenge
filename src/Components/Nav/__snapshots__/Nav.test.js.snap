// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Nav Component should match the snapshot 1`] = `
<div>
  <nav>
    <h1>
      NBA IS
      <img
        className="img-bball"
        src="basketball.png"
      />
      -Challenge
    </h1>
    <div
      className="btn-div"
    >
      <Link
        to="/rules"
      >
        <button
          className="btn"
        >
          Rules Expert
        </button>
      </Link>
      <Link
        to="/"
      >
        <button
          className="btn"
        >
          Home
        </button>
      </Link>
    </div>
  </nav>
</div>
`;
